modules/hrm/includes/class-announcement.php

```diff
@@ -196,6 +196,9 @@ function meta_boxes_cb( $post_id ) {
                         </select>
                     </td>
                 </tr>
+
+                <?php do_action( 'hr-announcement-table-last', $post ); ?>
```

Comment:
In every hooks or filters, we use the same separator as underscore `_`, not hyphen.